frontend/src/components/AppHeader/AppHeader.tsx

```diff
@@ -0,0 +1,19 @@
+import { AppHeaderWrapper } from './AppHeader.styled';
+import { PageHeader } from 'antd';
+
+export interface AppHeaderProps {
+  pageTitle?: string;
+  logo?: React.ReactNode;
+}
+
+export const AppHeader = ({ pageTitle, logo }: AppHeaderProps) => {
+  const providedLogo = logo ? logo : null;
```

Comment:
Zastanawiam się dlaczego takie konstrukcje tworzycie? Możesz mi opisać?

Reply:
nie potrawię wytłumaczyć :) może w przypadku js miałoby to większy sens ;)

Reply:
nie lepiej zrobić `{logo && <div>{logo}</div>}` zamiast właśnie taką konstrukcje gdzie w DOMie będzie pusty div